tools/debug_tools/src/debug_ui.rs

```diff
@@ -98,6 +59,12 @@ pub fn change_infotext(
             }
         }
 
-        text.sections[0].value = format!(" {fps:.1} fps, {frame_time:.3} ms/frame",);
+        if bools.dev_mode {
+            text.sections[0].value = if bools.show_fps_info {
+                 format!(" {fps:.1} fps, {frame_time:.3} ms/frame");
+            } else {
+                String::new();
+            };
```

Comment:
These semicolons have to be removed
```suggestion
        if bools.dev_mode {
            text.sections[0].value = if bools.show_fps_info {
                 format!(" {fps:.1} fps, {frame_time:.3} ms/frame")
            } else {
                String::new()
            };
```

Then the `if` returns a `String` value which can be assigned to the text section

Reply:
(That's on me, my suggestion was incorrect :D)